csharp/Platform.Setters/Setter[TResult, TDecision].cs

```diff
@@ -39,29 +48,29 @@ public Setter(TDecision trueValue, TDecision falseValue, TResult defaultValue)
         }
 
         /// <summary>
-        /// <para>Конструктор Setter делегирует свою работу более общему, добавив ему третий параметр со значением по умолчанию.</para>
-        /// <para>The Setter constructor delegates its work to a more general one by adding a third parameter with a default value.</para>
+        /// <para>Initializes a new instance of the Setter class with the third parameter set as default.</para>
+        /// <para>Иницаилизирует новый экземпляр класса Setter с третьим параметром, заданным по умолчанию.</para>
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Setter(TDecision trueValue, TDecision falseValue) : this(trueValue, falseValue, default) { }
         
         /// <summary>
-        /// <para>Initializes a new instance of the Setter class using the passed-in value as the default result value.</para>
-        /// <para>Инициализирует новый экземпляр класса Setter, используя переданное значение defaultValue в качестве результирующего по умолчанию.</para>
+        /// <para>Initializes a new instance of the Setter class with the <paramref name="defaultValue"/>.</para>
+        /// <para>Инициализирует новый экземпляр класса Setter, со значением <paramref name="defaultValue"/> в качестве результирующего по умолчанию.</para>
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Setter(TResult defaultValue) : base(defaultValue) { }
         
         /// <summary>
-        /// <para>Initializes a new instance of the Setter.</para>
+        /// <para>Initializes a new instance of the Setter class.</para>
         /// <para>Инициализирует новый экземпляр класса Setter.</para>
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Setter() { }
         
         /// <summary>
-        /// <para>Initializes a TDecision method that sets the passed-in value as the result value and returns trueValue.</para>
-        /// <para>Инициализирует TDecision метод, устанавливающий переданное значение в качестве результирующего и возвращающий значение trueValue.</para>
+        /// <para>Assigns the <paramref name="value"/> to the result and returns <see langword="true"/>.</para>
```

Comment:
My bad. Change `true` to the `true value`. Because a user defines its true value itself
`returns true value`